students/brgalloway/Lesson_9/mailroom/donor_models.py

```diff
@@ -0,0 +1,220 @@
+
+class Donor(object):
+   '''
+   Donor class takes a fullname to initiate a single donor. 
+   Each donor has three attributes: total donated, number of donations, and average donation
+   '''
+   def __init__(self, fullname):
+      self.fullname = fullname
+      self.donation_total = 0
+      self.times_donated = 0
+      self.average_donation = 0
+      
+   def __repr__(self):
+      # Prints out the object in dictionary format
+      # TODO output Donor object as dictionary to match representation
+      return f"{{\"{self.fullname}\": {{\"donation_total\": {self.donation_total:.2f}, \"times_donated\": {self.times_donated}, \"average_donation\": {self.average_donation:.2f}}}"
+       
+   @property
+   def donate(self):
```

Comment:
properties are usually nouns which means this should be named `donation_total`.. however since you're not "hiding" the attribute or doing anything special you really don't need to use a property